packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -0,0 +1,174 @@
+  const ExpansionTile({
+    Key key,
+    this.leading,
+    @required this.title,
+    this.backgroundColor,
+    this.onOpenChanged,
+    this.children: const <Widget>[],
+  }) : super(key: key);
+
+  /// A widget to display before the title.
+  ///
+  /// Typically a [CircleAvatar] widget.
+  final Widget leading;
+
+  /// The primary content of the list item.
+  ///
+  /// Typically a [Text] widget.
+  final Widget title;
+
+  /// Called when the tile expands or collapses.
+  ///
+  /// When the tile starts expanding, this function is called with the value
+  /// `true`. When the tile starts collapsing, this function is called with
+  /// the value false.
+  final ValueChanged<bool> onOpenChanged;
+
+  /// The widgets that are displayed when the tile expands.
+  ///
+  /// Typically [ListTile] widgets.
+  final List<Widget> children;
+
+  /// The color to display behind the sublist when expanded.
+  final Color backgroundColor;
+
+  @override
+  _ExpansionTileState createState() => new _ExpansionTileState();
+}
+
+class _ExpansionTileState extends State<ExpansionTile> with SingleTickerProviderStateMixin {
+  AnimationController _controller;
+  CurvedAnimation _easeOutAnimation;
+  CurvedAnimation _easeInAnimation;
+  ColorTween _borderColor;
+  ColorTween _headerColor;
+  ColorTween _iconColor;
+  ColorTween _backgroundColor;
+  Animation<double> _iconTurns;
+
+  bool _isExpanded = false;
+
+  @override
+  void initState() {
+    super.initState();
+    _controller = new AnimationController(duration: _kExpand, vsync: this);
+    _easeOutAnimation = new CurvedAnimation(parent: _controller, curve: Curves.easeOut);
+    _easeInAnimation = new CurvedAnimation(parent: _controller, curve: Curves.easeIn);
+    _borderColor = new ColorTween(begin: Colors.transparent);
+    _headerColor = new ColorTween();
+    _iconColor = new ColorTween();
+    _iconTurns = new Tween<double>(begin: 0.0, end: 0.5).animate(_easeInAnimation);
+    _backgroundColor = new ColorTween();
+
+    _isExpanded = PageStorage.of(context)?.readState(context) ?? false;
+    if (_isExpanded)
+      _controller.value = 1.0;
+  }
+
+  @override
+  void dispose() {
+    _controller.dispose();
+    super.dispose();
+  }
+
+  void _handleOnTap() {
+    setState(() {
+      _isExpanded = !_isExpanded;
+      if (_isExpanded)
+        _controller.forward();
+      else
+        _controller.reverse();
+      PageStorage.of(context)?.writeState(context, _isExpanded);
+    });
+    if (widget.onOpenChanged != null)
+      widget.onOpenChanged(_isExpanded);
+  }
+
+  Widget _buildChildren(BuildContext context, Widget child) {
+    return new Container(
+      decoration: new BoxDecoration(
+        color: _backgroundColor.evaluate(_easeOutAnimation),
+        border: new Border(
+          top: new BorderSide(color: _borderColor.evaluate(_easeOutAnimation)),
+          bottom: new BorderSide(color: _borderColor.evaluate(_easeOutAnimation))
+        )
+      ),
+      child: new Column(
```

Comment:
Yes. Now I'm a little surprised that everything worked without this, but it should have been there.